### PR TITLE
vep enum mapping updates

### PR DESCRIPTION
**Pull request by bpblanken**
This is probably partially complete, since I think we have more string cleanup to do here.

### CODE REVIEW

File: v03_pipeline/lib/annotations/shared.py [hunk 1]
```diff
@@ -8,6 +8,103 @@
...
+    'S',
+    'T',
+    'V',
+    'W',
+    'Y',
+    'X',
+    '*',
+    'U',
+]
+AMINO_ACIDS_LOOKUP = hl.dict(hl.enumerate(AMINO_ACIDS, index_first=False))
```
Comment (bpblanken) [inline v03_pipeline/lib/annotations/shared.py:36]: Not sure we even need this, but the mapping was straightforwards so I went ahead and did it.

File: v03_pipeline/lib/annotations/shared.py [hunk 2]
```diff
@@ -8,6 +8,103 @@
...
+
+BIOTYPES = [
```
Comment (bpblanken) [inline v03_pipeline/lib/annotations/shared.py:38]: Sourced from https://www.gencodegenes.org/pages/biotypes.html

The formats line up at least.

**Review by hanars — APPROVED** [review]
This all looks good. I also did confirm that we do not need all of  'lof',  'lof_filter', 'lof_flags', 'lof_info',and could instead use a much more compact rperesentation for those, but that might be out of thescope for this PR an warrant a different one

But for the record, we would need annotations for the following (names are proposals, you can make them whatever you want, and logic is very rough pseudo code):
- `is_lof_nagnag (bool): lof_flags === 'NAGNAG_SITE'`
- `lof_filters (array<str/enum>): (is_lof_nagnag() or lof == 'LC') and lof_filter and lof_filter.split(/&|,/)`

**Comment by bpblanken** [comment]
Cool, I'll just go ahead and make a separate ticket for the `is_lof_nagnag` and `lof_filters` annotations.

**Pull request merged**